### DISCUSSION TITLE
Loxone cloud connect

### DISCUSSION THREAD

**Discussion by FISystem** [JoDehli/PyLoxone#340] [Q&A]
Hello Guys!
Does anybody have any information for when the cloud connection is going to work?
Thanks for the answers!

**Comment by JoDehli** [comment]
@FISystem for now it is not planned to add it in the near future. You can enable local access. In my opinion it is the better solution.

**Reply by Kwull** [reply]
Hi @JoDehli, any news on cloud connection? I cannot use local access as HA and miniserver are on totally different networks. I have no access to the miniserver's local network, but can easily use cloud accessible apps and even web site.

**Reply by Kwull** [reply]
Maybe you can send me the vision what and where should be changed so I will try to implement it on my end

**Reply by Kwull** [reply]
I was able to successfully add miniserver via cloud connect via fixing a bit api.py. raise_if_not_200 should not raise an exception if status code is 307 (redirect). Will test if everything works at least on my end. Pull request to come then.

**Reply by Kwull** [reply]
Implemented in pull request #393

**Reply by JoDehli** [reply]
@Kwull I will take a look and test it with my setup with local access. I cannot test it for cloud access. I will merge it if it works as expected. Thanks for your help.
